docs/get_started/getting_started_linux_cmake.md

```diff
@@ -19,16 +19,16 @@ documented separately, as they require further setup.
 
 ### Install CMake
 
-IREE uses CMake version `>= 3.13`. First try installing via your distribution's
-package manager and verify the version:
+IREE uses CMake version `>= 3.13.4`. First try installing via your
+distribution's package manager and verify the version:
 
 ```shell
 $ sudo apt install cmake
-$ cmake --version # >= 3.13
+$ cmake --version # >= 3.13.4
 ```
 
 Some package managers (like `apt`) distribute old versions of cmake. If your
-package manager installs a version `< 3.13`, then follow the installation
+package manager installs a version `< 3.13.4`, then follow the installation
 instructions [here](https://cmake.org/install/) to install a newer version (e.g.
```

Comment:
I'll also note that I almost exclusively install ToT cmake and ninja via: `python -m pip install cmake ninja`. No muss, no fuss, works everywhere the same.